### ISSUE TITLE
Spelling: Game licensees

### ISSUE THREAD

**Issue by ihonen** [gbdev/pandocs#430]
So, there are a bunch of issues with the list of new licensees in 'The Cartridge Header':

- Incorrect capitalization (`kss` --> `KSS`, `pow` --> `Pow` etc.)
- Incorrect punctuation (`Pack in soft` --> `Pack-In-Soft`)

Then there's the list of old licensees that is not yet in the Pan Docs at all (instead, it is linked to). It's a real mess, with the name of almost every licensee being incorrectly spelled for whatever reason.

Incorrect spelling is easy enough to fix, but in some cases the correct spelling is not trivial to establish. In particular, there are two political decisions that need to be made:

1. Should the licensees be identified by their full names (`Tokuma Shoten Intermedia`) or their colloquial names (`Tokuma Shoten`)?
2. How should licensees that have since changed their names be identified? For example, `Ubi Soft` still exists but is now known as `Ubisoft`. On the other hand, `Hi Tech Entertainment` no longer exists but used to be called `Hi Tech Expressions`, and so on. Some companies may even have changed their names between the release of their first and last Game Boy games.

I'll gladly fix the spellings once we are clear on how to tackle the issue.

**Comment by ihonen** [comment]
My personal take:

1. I think it would be the better approach to identify licensees by their colloquial names.
2. Honestly, I don't know how to account for name changes. Maybe pick the name most prominently associated with the company in that era and mention the other names in footnotes?

**Comment by aaaaaa123456789** [comment]
I'd say that whatever the companies/groups call themselves (in media, documentation, etc.) is an adequate designation for them. Usually, for companies, this will be a short form of the full legal name of the company.

If some licensee has changed names since then, this could be indicated in a parenthetical or a footnote: "Ubisoft (Ubi Soft until 2003-09-09)".

**Comment by avivace** [comment]
> I'd say that whatever the companies/groups call themselves (in media, documentation, etc.) is an adequate designation for them. Usually, for companies, this will be a short form of the full legal name of the company.
> 
> If some licensee has changed names since then, this could be indicated in a parenthetical or a footnote: "Ubisoft (Ubi Soft until 2003-09-09)".

I agree with this. We could even link the Wikipedia pages as that one will always point to an updated entry

**Comment by ISSOtm** [comment]
+1 on the above, seems like consensus enough!

**Comment by quinnyo** [comment]
This looks like it's been pretty comprehensively completed at this point.
I think this can be closed.
